apps/web/components/Organization/Slice/Districts/DistrictsSlice.treat.ts

```diff
@@ -0,0 +1,35 @@
+import { style } from 'treat'
+import { themeUtils } from '@island.is/island-ui/theme'
+
+export const districtsTitle = style({
+  fontWeight: 600,
+  fontSize: 20,
+  lineHeight: 28 / 20,
+  ...themeUtils.responsiveStyle({
+    sm: {
+      fontSize: 24,
+      lineHeight: 34 / 24,
+    },
+  }),
+})
+
+export const districtsList = style({
+  columnGap: 30,
+  ...themeUtils.responsiveStyle({
+    xs: {
+      columnCount: 1,
+      marginTop: 40,
```

Comment:
could we use theme.spacing here for consistency?
```suggestion
      marginTop: theme.spacing[5],
```
same regarding the following margins, and try to follow the multiple of 8 like ex 75 could be rounded to 72 and use theme.spacing[9]